docs/src/library.md

```diff
@@ -51,3 +51,22 @@ Modules = [ManifoldDiff]
 Pages = ["riemannian_diff.jl"]
 Order = [:type, :function, :constant]
 ```
+
+## Proximal Maps
+
+Given a function ``f\colon \mathcal M \to \mathbb R``, its proximal map is defined
```

Comment:
```suggestion
Given a convex, lower semi-continuous function ``f\colon \mathcal M \to \mathbb R``, its proximal map is defined
```

Reply:
yes, but otherwise we usually use `f` for functions. I was thinking about changing the phi as well, but was not creative enough to come up with a new name for the current f.

Ah yes and I had a long day already so there are more typos in the docs again then I would have wished for, sorry for that.

Reply:
Hm, what about `d` like data?

Reply:
(instead of `f`/`p_f`)

Reply:
It's a point, so it should be `p`, `q` or something with more than one symbol. I really don't like introducing new non-standard single-letter variable names.

Reply:
That is exactly where I got stuck. p is the in put q the output and everything else looked too clumsy or too easy to be confused with the input p.

Reply:
How is `p_f` too clumsy? You can just use subscript in math and it's clear what is going on.

Reply:
Or it could be `p_data` for less confusion with the input maybe?

Reply:
WE could take `r` that is still close by to p and q but not a neighbouring letter of p?

Reply:
:disappointed:

Reply:
Hm, not your favourite? I feel that `p` and `p_f` can too easily be confused,

Reply:
Ok, `p_data` would be fine with me as well, since (compared to f) we say explicitly that that is fixed data that way :)

Reply:
Great, thanks :slightly_smiling_face: